### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by jhaaaa**

### CODE REVIEW

File: docs/dev-concepts/faq.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Currently, XMTP Labs (the company) operates all of the network nodes in the two
 
 Decentralization of the XMTP network will be achieved by a diverse set of independent third parties operating nodes.
 
+<!--do we have an updated timeline for this shareout?-->
```
Comment (jhaaaa) [inline docs/dev-concepts/faq.md:34]: @mkobetic @snormore Any thoughts on the timeline for sharing a decentralization roadmap? =)

File: docs/dev-concepts/faq.md [hunk 2]
```diff
@@ -87,21 +88,25 @@ There are no message storage and retrieval-related fees incurred by developers f
 
 ### How does XMTP establish a secure and fraud-proof relationship between two identities?
 
-Blockchain accounts sign and advertise a set of keys that XMTP uses to establish a shared secret for encryption using another blockchain account’s keys. These keys attest to the authenticity of both accounts and are required to add messages to their conversation. No third-party apps or relayers are involved in this process.
+Blockchain accounts sign and advertise a set of keys to start using XMTP. These keys attest to the authenticity of both accounts. XMTP uses these keys to establish a shared secret between the blockchain accounts and uses it to generate a key used to encrypt an invitation that allows the blockchain accounts to start exchanging messages. No third-party apps or relayers are involved in this process.
```
Comment (jhaaaa) [inline docs/dev-concepts/faq.md:91]: @mkobetic I made these refinements to reflect XMTP V2 - any thoughts?

File: docs/dev-concepts/faq.md [hunk 3]
```diff
@@ -87,21 +88,25 @@ There are no message storage and retrieval-related fees incurred by developers f
...
+Yes. Each blockchain account address is represented by an XMTP identity key. This identity key is a part of a key bundle that only the address that generated it can use to authenticate messages.
 
-To learn more about XMTP identities, see [Sign to send and receive messages using apps built with XMTP](account-signatures).
+To learn more about XMTP identity keys, see [Key generation and usage in XMTP](key-generation-and-usage).
 
 ### Do apps built with XMTP need to decrypt messages with blockchain account private keys each time?
 
-When a user starts a new messaging session with an app built with XMTP, the user must sign with their blockchain account private key to decrypt their XMTP key bundle, which is then used for message decryption. A one-time signature is also required to create that key bundle, which includes an XMTP identity.
+When a user starts a new messaging session with an app built with XMTP, the user must sign with their blockchain account private key to decrypt their XMTP key bundle, which is then used for invitation and message decryption. A one-time signature is also required to create that key bundle, which includes an XMTP identity.
```
Comment (jhaaaa) [inline docs/dev-concepts/faq.md:105]: @mkobetic I made these refinements to reflect XMTP V2 - any thoughts?

File: docs/dev-concepts/faq.md [hunk 4]
```diff
@@ -179,6 +187,10 @@ The XMTP SDK currently requires you to use [ethers](https://ethers.org/) or anot
 
 ### Why does my app request a new signature for XMTP when I refresh my browser?
 
-The XMTP client provided by the SDK requires a user's signature in order to decrypt their XMTP message encryption keys. This process must be repeated when starting a new session since there is no secure place in the browser to persist decrypted keys. Based on developer and community feedback, we are researching more robust approaches to secure key management.
+The XMTP client provided by the SDK requires a user's signature to decrypt their keys needed for invitation and message encryption. This process must be repeated when starting a new session since there is no secure place in the browser to persist decrypted keys. 
+
+If you are using the JavaScript client SDK, you might consider [manually handling private key storage](/docs/client-sdk/javascript/tutorials/quickstart#manually-handling-private-key-storage), but only with the understanding that this approach **requires the utmost care**.
+
```
Comment (jhaaaa) [inline docs/dev-concepts/faq.md:193]: @neekolas - what do you think of this update to the FAQ? Is this something we might want to surface here - or is it better that it remain in the README context only?

File: docs/dev-concepts/faq.md [hunk 5]
```diff
@@ -31,6 +31,7 @@ Currently, XMTP Labs (the company) operates all of the network nodes in the two
 
 Decentralization of the XMTP network will be achieved by a diverse set of independent third parties operating nodes.
 
+<!--do we have an updated timeline for this shareout?-->
 XMTP Labs is working toward a phased decentralization of the network and will share a roadmap in the coming months.
```
Comment (jhaaaa) [inline docs/dev-concepts/faq.md:35]: @mkobetic @snormore Any thoughts on the timeline for sharing a decentralization roadmap? =)

File: docs/dev-concepts/faq.md (hunk 2 above)
Comment (mkobetic) [reply docs/dev-concepts/faq.md:91]: I would drop the `These keys attest to the authenticity of both accounts.`. It's not clear what `both accounts` means and doesn't seem necessary. I would also split the long sentence in the middle, it's a bit long to follow.

File: docs/dev-concepts/faq.md (hunk 3 above)
Comment (mkobetic) [reply docs/dev-concepts/faq.md:105]: The change is OK, but my answer to the question would be "No, the account key is only used for signing". The phrasing of the question may be a bit off, what did we want to convey with this one?

Comment (jhaaaa) [reply docs/dev-concepts/faq.md:105]: Adjusted text per chat with Martin.

File: docs/dev-concepts/faq.md [hunk 6]
```diff
@@ -169,16 +180,10 @@ Have other questions or ideas about group messaging? Post to the [XMTP discussio
...
-The XMTP SDK is written in TypeScript for JavaScript and TypeScript projects. It can be used with browser-based frontend frameworks like React and in Node.js.
+The XMTP SDK is [available for multiple languages](/sdks-and-tools#sdks), including JavaScript, Flutter, and Swift.
 
 Have other questions or ideas for future language or environment support? Post to the [XMTP discussion forum](https://github.com/orgs/xmtp/discussions).
 
 ### Which web3 libraries does the XMTP SDK require?
 
 The XMTP SDK currently requires you to use [ethers](https://ethers.org/) or another web3 library capable of supplying an [ethers Signer](https://docs.ethers.io/v5/api/signer/), such as [wagmi](https://wagmi.sh/).
-
-### Why does my app request a new signature for XMTP when I refresh my browser?
```
Comment (jhaaaa) [inline docs/dev-concepts/faq.md:180]: Removed this question and combined concepts in https://github.com/xmtp/xmtp-dot-org/pull/270/files#diff-3de10b02d0ab6fa186d4ca1f14dff1c2c8520ec6055e394ae850dcb9f959ece1R102. 

I think they are different ways of asking the same question?

File: docs/dev-concepts/faq.md [hunk 7]
```diff
@@ -87,21 +87,29 @@ There are no message storage and retrieval-related fees incurred by developers f
...
+
+### Why do apps built with XMTP require a user to sign with their blockchain account private keys each time they start a new messaging session?
+
+When a user starts a new messaging session, they must sign with their blockchain account private key to decrypt their XMTP key bundle. The key bundle is then used for invitation and message decryption.
 
-To learn more about XMTP identities, see [Sign to send and receive messages using apps built with XMTP](account-signatures).
+Because there is no secure place in the browser to persist a decrypted key bundle, the app can use the bundle for the current session only. Once the user starts a new session, such as after refreshing their browser, they must sign again to decrypt their key bundle.
 
-### Do apps built with XMTP need to decrypt messages with blockchain account private keys each time?
+If you are using the JavaScript client SDK, you might consider [manually handling private key storage](/docs/client-sdk/javascript/tutorials/quickstart#manually-handling-private-key-storage), but only with the understanding that this approach **requires the utmost care**.
```
Comment (jhaaaa) [inline docs/dev-concepts/faq.md:108]: @neekolas Just opening a new comment after relocating this content. What do you think of surfacing this private key storage info here?  Is it OK - or should we keep it in the full context of the README?

Comment (neekolas) [reply docs/dev-concepts/faq.md:108]: I think it's fine for now. With version 8 of the SDK, all these instructions are going to have to change. I should have more details to share soon. 

But the gist is that private keys are not always going to be accessible

Comment (jhaaaa) [reply docs/dev-concepts/faq.md:108]: Thank you so much for taking a look, Nick - and got it - understood that the instructions will change for v8. I appreciate the heads up, as always!

**Pull request merged**